internal/credentials/credentials.go

```diff
@@ -1,8 +1,5 @@
 // +build windows
 
-// Package credentials holds the necessary structs and functions for adding
```

Comment:
Are we losing our package doc comment? Those are really nice to have.

Reply:
haha commented the same below. Must have gotten removed during rebase

Reply:
Adding back

Reply:
@kevpar done